insights/parsers/cpuset_cpus.py

```diff
@@ -0,0 +1,51 @@
+"""
+InitProcessCgroup - File ``/sys/fs/cgroup/cpuset/cpuset.cpus``
+==============================================================
+
+This parser reads the content of ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
+This file shows the default cgroup cpuset.cpu of system. The format
+of the content is string including comma.
+"""
+
+from .. import parser, CommandParser, LegacyItemAccess
+from insights.specs import Specs
+
+
+@parser(Specs.cpuset_cpus)
+class CpusetCpus(CommandParser, LegacyItemAccess):
```

Comment:
`LegacyItemAcess` does not work when `self.data` is not a `dict`. Better remove it.